src/controllers/index.js

```diff
@@ -10,4 +10,4 @@ const getUsers = async (req, res, next) => {
     }
 }
 
-module.exports = { getUsers }
+module.exports = { getUsers, login: require("./login") }
```

Comment:
Very cool! Saves having to declare the login variable again. I was wondering if this `dir/index.js` pattern was gonna be not worth the trouble since we're not using ES6 anymore. But this method takes care of that 😎